src/util.ts

```diff
@@ -192,6 +199,32 @@ export function getRuntimeTrackingString(): string {
   }
 }
 
+export function getDirName() {
+  let dirToUse = '';
+  try {
+    dirToUse = __dirname;
+  } catch (e) {
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    // @ts-ignore
+    dirToUse = path.dirname(fileURLToPath(import.meta.url));
```

Comment:
This seems to only be relative to the current file rather than the caller's directory, which is fine but worth noting. [Optional] Alternatively we could point this to the 'root' of the module (or `src`) for consistency and rename this function appropriately.